src/instructlab/model/backends/backends.py

```diff
@@ -135,9 +136,14 @@ def determine_backend(model_path: pathlib.Path) -> str:
         ) from e
 
     if is_gguf:
+        logger.debug(f"Model is a GGUF file, using {LLAMA_CPP} backend.")
         return LLAMA_CPP
 
-    raise ValueError(f"The model file {model_path} is not a GGUF format. Unsupported.")
+    raise ValueError(
+        f"The model file {model_path} is not a GGUF format nor a directory. Cannot determine which backend to use. \n"
+        f"Please use a GGUF file for {LLAMA_CPP} or a directory containing tensor files for {VLLM}. \n"
```

Comment:
should this be more explicit and say "..a directory containing huggingface safetensors files..." instead of just "tensor files" ?